src/Admin/ActivationRedirect.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Helper class for handling the redirection to the onboarding page.
+ *
+ * @package Automattic\WooCommerce\Pinterest\Admin
+ * @since   X.X.X
```

Comment:
All of these placeholders should be lowercase. I won't flag them all, but they should be updated in all of these files.

```suggestion
 * @since   x.x.x
```